.github/workflows/pr-build.yml

```diff
@@ -68,8 +68,23 @@ jobs:
         key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
         restore-keys: |
           ${{ runner.os }}-node-
-    - name: test
+    - name: Test CDK
       working-directory: cdk_infra/
       run: |
         npm install
         npm run test
+  adot-testbed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '17'
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: adot-testbed build
+        working-directory: adot-testbed/
+        run: gradle build -x test
```

Comment:
is the intent to run the testbed tests here?

Reply:
In this PR build I am excluding the test part. I think we wont be needing creds for the build part. just to catch for any build failures

Reply:
makes sense.

Reply:
This is an interesting idea though. Should we add a workflow in this repository to run the tests against the latest version of the collector? It would act as an integration test. I wish we could do it at PR time but I wouldn't love introducing credentials into the PR workflow.